image-customize

```diff
@@ -88,18 +88,23 @@ class InstallAction(ActionBase):
 
 
 class BuildAction(ActionBase):
-    '''Build and install distribution package(s) from dist tarball'''
+    '''Build and install distribution package(s) from dist tarball or source RPM'''
 
     @staticmethod
     def execute(machine_instance, source):
         # upload the tarball or srpm
         sourcename = os.path.basename(source)
+
+        if source.endswith(".src.rpm"):
+            version = None  # srpm knows its own version
```

Comment:
Yes, that's been on my todo list since the original PR that introduced `--build`. c-podman/machine/starter-kit don't actually produce a .tarball file, so we can't use that -- but it seems easier to just update debian/changelog and PKGBUILD right away there, then I can simplify the `else:` branch here and remove the seddery from `build_{arch,deb}`.

Reply:
I sent https://github.com/cockpit-project/cockpit-podman/pull/909 for this. Once that gets approved and lands, I'll apply the same thing to cockpit-machines, then I'll drop the VERSION/SOURCE seddery here.

Reply:
And https://github.com/cockpit-project/cockpit-machines/pull/557 for exactly the same change for machines.

Reply:
Once this lands, we can [land this commit](https://github.com/martinpitt/bots/commit/871ac599263ae6f301f42c667a556547112c60a1) to drop the seddery from image-customize.

Reply:
Done in #2902, I'll rebase this on top of that.